WORKSPACE

```diff
@@ -420,3 +420,21 @@ load("//rules/rbe_repo:util.bzl", "rbe_autoconfig_root")
 # Needed for testing purposes. Creates a file that exposes
 # the value of RBE_AUTOCONF_ROOT
 rbe_autoconfig_root(name = "rbe_autoconfig_root")
+
+# Pull in dependencies of base_images_docker.
+load("@base_images_docker//package_managers:repositories.bzl", package_manager_deps = "deps")
+
+package_manager_deps()
+
+load("@io_bazel_rules_python//python:pip.bzl", "pip_import", "pip_repositories")
```

Comment:
lets try to move all this to https://github.com/GoogleContainerTools/base-images-docker/blob/master/package_managers/repositories.bzl

Reply:
Mentioning why this wasn't done in a top level comment for visibility.